modules/rules_itest/0.0.11/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(
+    name = "rules_itest",
+    version = "0.0.11",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "aspect_bazel_lib", version = "1.42.0")
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_go", version = "0.44.0")
+
+# TODO(zbarsky): This can't be marked dev_dependency, otherwise `examples` module won't build. Why?
+bazel_dep(name = "gazelle", version = "0.34.0")
```

Comment:
[this is now fixed ](https://github.com/dzbarsky/rules_itest/commit/873b870ed5645966ce17ba2977079d53994e0406), it's because the `itest.bzl` in the root triggered an eager load of gazelle in that BUILD file. The error when this happens isn't great :/